### ISSUE TITLE
Clearer language about Retention Policy "delete" actions with respect to Exchange Online

### ISSUE THREAD

**Issue by LukeAllinson-NS** [MicrosoftDocs/microsoft-365-docs#4326]
Hello; another one from me I'm afraid.

In relation to the "Principles of Retention" (from this page https://docs.microsoft.com/en-us/microsoft-365/compliance/retention?view=o365-worldwide), can I ask if it is possible to make some of the language clearer to better explain the behaviour of Retention Policy "delete" actions? Particularly when concerned with Exchange messages. I ask this as I've been setting up some complex policies and labels both in my demo environment and for a customer and the behaviour I have observed does not match what is documented (depending on what you class as a "delete"). 

For example, referring to the complex scenarios:

> More complex examples that combine retain and delete actions:
> An item has the following retention settings applied to it:
> - A retention policy for delete-only after five years
> - A retention policy that retains for three years and then deletes
> - A retention label that retains-only for seven years
>
> Outcome: The item is retained for seven years because retention takes precedence over deletion and seven years is the longest retention period. At the end of this retention period, the item is deleted because of the delete action from the retention policies that was deferred while the item was retained.
> Although the two retention policies have different dates for the delete actions, the earliest the item can be deleted is at the end of the longest retention period, which is longer than both deletion dates. In this example, there is no conflict to resolve for the deletion dates so all conflicts are resolved by the second level.
> 
> An item has the following retention settings applied to it:
> - An unscoped retention policy that deletes-only after ten years
> - A scoped retention policy that retains for five years and then deletes
> - A retention label that retains for three years and then deletes
> 
> Outcome: The item is retained for five years because that's the longest retention period. At the end of that retention period, the item is deleted because of the delete action of three years from the retention label that was deferred while the item was retained. Deletion from retention labels takes precedence over deletion from all retention policies. In this example, all conflicts are resolved by the third level.

In the first example (when applied to Exchange mailboxes), I have observed that an affected item will be moved to the Recoverable Items folder at the end of the shortest deletion period; in this case 3 years. The item would be removed from Recoverable Items after the "RetainDeletedItemsFor" parameter on the mailbox (default 14 days) and would then be recoverable via eDiscovery up until the end of the retention period; in this case 7 years.

In the second example (again, when applied to Exchange mailboxes), items older than 3 years would be moved to recoverable items (and then emptied after 14 days) then permanently deleted (i.e. unrecoverable) after 5 years.

I appreciate that I'm only talking about exchange here, however the documentation's definition of "delete" is to permanently remove the item from the mailbox, however the Delete action does move the item to the Recoverable items folder even if the item is within a longer retention period. From a user's perspective, this is still a deletion as the item is removed from the inbox.

The Exchange-specific page (https://docs.microsoft.com/en-us/microsoft-365/compliance/retention-policies-exchange?view=o365-worldwide) does mention that items are moved to recoverable items when the deletion period ends, but it doesn't mention what happens when the deletion period is shorter than the retention period.

Apologies for the length of this; I hope I've made this sufficiently clear. Please let me know if you would like me to provide any further information or evidence of this.

Many Thanks
Luke

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6c49609d-3a62-b9be-24bf-f8fa6948d390
* Version Independent ID: df547c86-fbc7-4f87-e893-49d07b045373
* Content: [Learn about retention policies & labels to automatically retain or delete content - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/retention?view=o365-worldwide#retention-labels)
* Content Source: [microsoft-365/compliance/retention.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/retention.md)
* Service: **o365-seccomp**
* GitHub Login: @cabailey
* Microsoft Alias: **cabailey**

**Comment by yogkumgit** [comment]
@cabailey Please add your comment and help us in resolving this issue. Thanks.

**Comment by cabailey** [comment]
@LukeAllinson-NS  In the context of the principles of retention, where we're addressing how conflicts get resolved, delete means eligible for permanent deletion (so not searchable by eDiscovery), regardless of where that content currently is.  So for Exchange, that content could be in the Recoverable Items folder but won't be permanently deleted from there while it is subject to retention from another policy.  I'll have a look at clarifying this in the docs.

**Comment by LukeAllinson-NS** [comment]
Great, thanks.
As I said, it is clear from my testing/experience what "delete" means in the context of Exchange, but it would be super to see a note somewhere about the "move to recoverable items" action (i.e. Exchange hard-delete) occurring even if a retention is applied.

**Comment by cabailey** [comment]
Noted - created a work item to incorporate this clarification into the docs.  Although you're asking for it in the context of Exchange, it will need adding for all workloads, so not a quick fix.
